docs/log-in-as-po.md

```diff
@@ -1,24 +1,55 @@
 # Log in as public officer
 
-1. Go to the [Docs portal](https://docs.developer.tech.gov.sg/) and click **Log in with TechPass**.
-<kbd>![log-in-with-techpass](assets/images/access-sgts-services-using-techpass/first.png)</kbd>
-2. Sign in to your TechPass account or choose your TechPass account.
-<kbd>![sign-in](assets/images/access-sgts-services-using-techpass/log-in-with-techpass.png)</kbd>
+Public officer can log in to TechPass using your non-SE GSIB device or your Government Managed Device (GMD).
+
+**To log in with a non-SE GSIB device**
+
+1. Go to the [Docs portal](https://docs.developer.tech.gov.sg/) and click **Log in**.
 
-If you are signing in from your GMD, you will be prompted to authenticate your WOG account.
+<kbd>![new-mfa](assets/images/onboarding/po-non-se/new-docs-portal.png)</kbd>
 
-3. Enter the verification code displayed for your SG Govt M365 profile on your Authenticator app.
+2. Click **Log in with TechPass**.
 
-<kbd>![verify-wog-for-po](assets/images/access-sgts-services-using-techpass/verification-code-po.png)-->
+<kbd>![new-mfa](assets/images/onboarding/po-non-se/new-docs.png)</kbd>
```

Comment:
<img width="301" alt="image" src="https://user-images.githubusercontent.com/11743256/224989395-b83d723e-47e6-47d9-865b-a268487047b3.png">

Reply:
sorry, it appeared fine on local preview, will fix this, same goes for the rest of text.

Reply:
i mean, images.